bin/node/runtime/src/lib.rs

```diff
@@ -537,24 +548,44 @@ impl_runtime_apis! {
 
 	impl sp_bridge_eth_poa::EthereumHeadersApi<Block> for Runtime {
```

Comment:
We might need separate runtime API for each bridge or we might pass bridge instance name as a parameter. That aligns with the `BridgeHub` pallet idea, which could be a unified interface for all bridge instances.